src/Tracking/TrackingData/DonationData.php

```diff
@@ -38,12 +37,16 @@ private function getFirstDonationDate() {
 
 		$date = $wpdb->get_var(
 			"
-				SELECT post_date_gmt
-				FROM {$wpdb->posts}
-				WHERE post_status IN ({$this->getDonationStatuses()})
-				ORDER BY post_date_gmt DESC
-				LIMIT 1
-				"
+			SELECT post_date_gmt
+			FROM {$wpdb->posts} as p
+			INNER JOIN {$wpdb->donationmeta} as dm
+			ON p.id=dm.id
+			WHERE post_status IN ({$this->getDonationStatuses()})
+			AND dm.meta_key='_give_payment_mode'
+			AND dm.meta_value='live'
```

Comment:
@JasonTheAdams Change this to `test` when test locally.

Reply:
@JasonTheAdams In the morning, I will confirm that does `_give_payment_mode ` meta key stores in renewal or not. If we store this meta key in renewal donation meta then the result will be correct.

Reply:
I find out that we do not store `_give_payment_mode ` meta key for renewal payment. This means renewal payment will be ignored when calculating donation metrics. Need to add separate SQL keep to handle renewal payment. Let chat about this.

Reply:
We have a similar issue open for report https://github.com/impress-org/givewp/issues/5398

Reply:
I will create an issue where we can continue the discussion.